### PR TITLE
Add a link to the related KB article

### PR DESCRIPTION
**Pull request by PiotrekKrakowiak**
Add a link to the KB article describing a scenario where search results are not available for the users who have permissions to see the content.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointOnline/make-site-content-searchable.md [hunk 1]
```diff
@@ -25,6 +25,8 @@ When users search on a site, results can come from many places such as columns,
   
 > [!NOTE]
 > Search results are always security trimmed, so users will only see content they have permission to see. The search settings only define what content is included in the search index.
+> 
+> Please be aware that there are specific scenarios when the users have permissions to see the content but are still unable to find it in the search results. Refer to the article: https://docs.microsoft.com/en-us/sharepoint/troubleshoot/search/search-results-dont-appear-for-group-owners for more details.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/make-site-content-searchable.md:29]:
```suggestion
> There are specific scenarios where users have permissions to see the content but are still unable to find it in the search results. For more details, see [Search results don't appear for group owners after creating a new Office 365 group](/sharepoint/troubleshoot/search/search-results-dont-appear-for-group-owners).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @PiotrekKrakowiak, kindly commit the suggested changes. Many thanks.

**Comment by PiotrekKrakowiak** [comment]
commiting changes

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by PiotrekKrakowiak** [comment]
Hi Johan,

any idea when this will be published?

Thanks!
Piotrek

________________________________

**Comment by scanum** [comment]
@SerdarSoysal PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by PiotrekKrakowiak** [comment]
I am a bit unsure where we are stuck here.... is there still anything that I need to click? 🙂 I have a customer impatiently waiting for this update....

Thanks!
Piotrek

________________________________

**Pull request merged**